src/test/java/bookmarked/command/ListCommandTest.java

```diff
@@ -30,8 +32,9 @@ public void listCommand_emptyList_exceptionThrown(){
     @Test
     public void listCommand_emptyArguments_exceptionThrown(){
         ArrayList<Book> listOfBooks = new ArrayList<>();
+        ArrayList<User> listOfUsers = new ArrayList<>();
 
-        ListCommand listDefault = new ListCommand(listOfBooks, "list /sortby");
+        ListCommand listDefault = new ListCommand(listOfBooks, "list /sortby", listOfUsers);
```

Comment:
maybe can add test case for input like:
- `"list /sortby     "`, 
- `"list ////sortby"`, 
- `"list /unknown something"`, 
- `"list /unknown"`,
- `"list"`,
- `"list   "`,
- `"list argument"`,
?